CHANGES.md

```diff
@@ -5,6 +5,8 @@
 ### Features
 
 * ancestral: Add `--seed` argument to enable deterministic inference of root states by TreeTime. [#1690][] (@huddlej)
+* export v2: The string "none" is now an invalid value for `--color-by-metadata` and `--metadata-columns` options to prevent clashes with Auspice's internal use of "none". [#1113][] (@joverlee521)
+* schema: The string "none" is now an invalid branch label, node_attr key, and coloring key. [#1113][] (@joverlee521)
```

Comment:
Are these technically https://github.com/nextstrain/augur/labels/breaking changes to be put in the "Major Changes" section?

Reply:
Ah yes! Thanks for catching that. Will rebase and update changelog after latest release is complete.